contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+# UI/UX
+
+Author: [Ang Shi Ya](https://github.com/AngShiYa)
+
+## Overview
+
+User Interface (UI) and User Experience (UX) are terms often used interchangeably. In fact, These two terms are quickly becoming dangerous buzzwords that are used inaccurately, creating massive confusion in the industry. A simple google search will show that UI Designer and UX Designer jobs are listed under similar titles and descriptions. 
+
+Learning the key differences between UI and UX can not only help us identify and settle down for the correct job role, but can also help us apply each of these concepts appropriately to our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
+
+## User Interface vs User Experience
+
+UI is about the design of the buttons, the layout of the website and the responsiveness of the application. It encompasses the presentation and interactivity of the application. On the other hand, UX is concern with the ease of use and the pleasure provided through using the application. In short, UI is about the look and feel while UX is about customer satisfaction.
```

Comment:
Appearance might not fully capture what UI is since interaction is also in the picture. Experience might be a little weird, since in the full form it will translate to user experience is about experience.